### PR TITLE
Add Rotation property

### LINKED ISSUE #1082: Allow programmatic rotation of map
[From the forum](https://groups.google.com/d/msg/mitappinventortest/mLx4fr1E36A/6dH4NqndBgAJ): Users can rotate the map using a two-finger gesture, but there is no property in the blocks to get/set the rotation.

### PR DESCRIPTION
**Pull request by ChitiSims**
Users could rotate the map but not programmatically. This change adds
blocks to allow developers to programmatically change the map rotation.

Fixes #1082

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/util/NativeOpenStreetMapController.java [hunk 1]
```diff
@@ -1168,6 +1168,16 @@ public int getOverlayCount() {
     return view.getOverlays().size();
   }
 
+  @Override
+  public void setRotation(float Rotation) {
+	  view.setMapOrientation(Rotation);
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/util/NativeOpenStreetMapController.java]: Replace tab character with spaces.

File: appinventor/components/src/com/google/appinventor/components/runtime/util/NativeOpenStreetMapController.java [hunk 2]
```diff
@@ -1168,6 +1168,16 @@ public int getOverlayCount() {
...
+  }
+
+  @Override
+  public float getRotation() {
+	  return view.getMapOrientation();
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/util/NativeOpenStreetMapController.java]: Replace tab character with spaces

**Comment by jisqyv** [comment]
MERGED from Gerrit

**Comment by Amerkashi** [comment]
The versioning.js doesn't need update?

**Comment by ewpatton** [comment]
@Amerkashi The version I put on Gerrit includes the versioning change. See the diff at c0cd37b

**Comment by Amerkashi** [comment]
@ewpatton Thanks

**Pull request closed without merging**